src/components/home/Testimonial.vue

```diff
@@ -52,26 +52,30 @@
 </template>
 
 <script>
-  export default {
-    name: "Testimonial",
-  };
+export default {
+  name: "Testimonial"
+};
 </script>
 
 <style lang="sass" scoped>
+.testimonial-card
```

Comment:
I don't think you should reindent everything, this is what broke the font too. It was indented, so it would be nested under the image (`.quotes`)